package.json

```diff
@@ -22,7 +22,7 @@
     "repl": "yarn start --entryFile repl",
     "gen-id": "ts-node -Te \"let g = require('./src/common/generate-id').generateId; for (let i = 0, l = process.argv[1] || 1; i < l; i++) { g().then(console.log) }\"",
     "edgedb:gen": "node --loader ts-node/esm/transpile-only --experimental-loader=extensionless src/core/edgedb/generator/run.ts",
-    "edgedb:seed": "cat dbschema/seeds/*.edgeql | edgedb",
+    "edgedb:seed": "node --loader ts-node/esm/transpile-only --experimental-loader=extensionless src/core/edgedb/seeds.run.ts",
```

Comment:
Not looking for a change, but UGH, this is where I _**love**_ bun...

Reply:
Yeah I want to get there too. We need to refactor the TS keyof transformer first. Currently we need TS to compile our code to get that transformation done. Once that's done, we can compile/transpile with another faster tool.